kumascript/macros/EmbedLiveSample.ejs

```diff
@@ -14,7 +14,7 @@
 //
 // See also : LiveSampleLink
 
-var sampleId = $0;
+var sampleId = $0 || $token.location.start.offset;
```

Comment:
❤️ I like how you decided to pass the `token` as part of the macro's execution context.